python/paddle/fluid/tests/unittests/test_zero_dim_tensor.py

```diff
@@ -232,31 +234,32 @@ def test_static(self):
     {'func': paddle.multiply, 'cls_method': '__mul__'},
     {'func': paddle.divide, 'cls_method': '__div__'},
     {'func': paddle.subtract, 'cls_method': '__sub__'},
-    paddle.pow,
-]
-
-binary_api_list_without_grad = [
+    {'func': paddle.pow, 'cls_method': '__pow__'},
     {'func': paddle.add, 'cls_method': '__add__'},
     {'func': paddle.subtract, 'cls_method': '__sub__'},
     {'func': paddle.multiply, 'cls_method': '__mul__'},
     {'func': paddle.divide, 'cls_method': '__div__'},
     {'func': paddle.subtract, 'cls_method': '__sub__'},
-    paddle.pow,
-    {'func': paddle.mod, 'cls_method': '__mod__'},
-    paddle.floor_mod,
-    paddle.remainder,
+    {'func': paddle.pow, 'cls_method': '__pow__'},
+]
+
+binary_api_list_without_grad = [
     {'func': paddle.equal, 'cls_method': '__eq__'},
     {'func': paddle.not_equal, 'cls_method': '__ne__'},
     {'func': paddle.greater_equal, 'cls_method': '__ge__'},
     {'func': paddle.greater_than, 'cls_method': '__gt__'},
     {'func': paddle.less_equal, 'cls_method': '__le__'},
     {'func': paddle.less_than, 'cls_method': '__lt__'},
+    {'func': paddle.remainder, 'cls_method': '__mod__'},
+    paddle.mod,
+    paddle.floor_mod,
     paddle.logical_and,
     paddle.logical_or,
     paddle.logical_xor,
```

Comment:
shall we add `paddle.bitwise_and/or/xor` ?

Reply:
Done